src/actions/actions.js

```diff
@@ -2,17 +2,14 @@ import nba from 'nba-api-client';
 
 export const fetchPlayers = () => {
     return async (dispatch) => {
-        const response = await nba.leaguePlayerGeneralStats({TeamID: 1610612761, Season: "2019-20"}).then(function(data) {
-            const dataArray = Object.values(data.LeagueDashPlayerStats);
-            return dataArray;
-        });
-
-        dispatch({ type: 'FETCH_PLAYERS', payload: response});
+        const response = await nba.leaguePlayerGeneralStats({TeamID: 1610612761, Season: "2019-20"}).then((data) => Object.values(data.LeagueDashPlayerStats));
```

Comment:
So I was playing around with the API endpoints last night and realized that `leaguePlayerGeneralStats` endpoint does not give any general information (like jersey number, position), just stats.

I will create another ticket to change this once this one is merged in and in that ticket let's change the endpoint to use `nba.teamRoster` . That gives us coaches and players general info